packages/flutter_tools/templates/app/lib/main.dart.tmpl

```diff
@@ -36,6 +36,10 @@ class MyApp extends StatelessWidget {
         // Notice that the counter didn't reset back to zero; the application
         // is not restarted.
         primarySwatch: Colors.blue,
+        // This makes the visual density adapt to the platform that you run
+        // the app on. For desktop platforms, the controls will be smaller and
+        // closer together (more dense) than on mobile platforms.
+        visualDensity: VisualDensity.adaptivePlatformDensity,
```

Comment:
why do we need this? isn't being adaptive the default?

Reply:
No, being adaptive isn't the default, because the Material Design team wanted the "mobile" density to be the default.  Their reasoning was that some desktop systems have touch screens, and so those screens should have 48x48 touch targets, so as not to compromise accessibility.

We had anecdotal evidence, however, that developers would like the option to have it be adaptive, so this was the compromise that we came up with.